PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+JIRA: https://jira.dev.bbc.co.uk/browse/YOUR-TICKET
+
+_PR changes summary to go here._
+
+- [ ] JIRA ticket added
+- [ ] Pull request URL added to JIRA ticket
```

Comment:
@sareh  Is there not an existing way to integrate JIRA and have this and the above line be automatic, e.g. PR contains jira ticket ref and a script writes to ticket automatically?

Reply:
Yes there is. I asked @nathangochin last week whether he could have a look at this - it requires an admin I think.

Reply:
When you name your branch or PR with the JIRA ticket, it should automatically show up in JIRA. For more details to be shown on the ticket, e.g. merge status, I think we'd need this OAuth set up:
https://confluence.atlassian.com/adminjiracloud/connect-jira-cloud-to-github-814188429.html
I don't have admin permissions on JIRA or Github for this.